### PR TITLE
Wallix

### PR DESCRIPTION
**Pull request by yoennb**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by edik24** [comment]
Hi @yoennb,
can you please [sign CLA](https://github.com/demisto/content/pull/25840#issuecomment-1508222422),
and fill the [contribution form](https://github.com/demisto/content/pull/25840#issuecomment-1508224125)?

**Comment by yoennb** [comment]
We are working on it. We first need to finalize the video required to sign the contribution form.

**Comment by samuelFain** [comment]
Hi @yoennb, thank you for contributing!
I'm currently reviewing your PR and should have a review ready shortly.
@ShirleyDenkberg, can you please review?

File: Packs/WALLIXBastion/Integrations/WAB/WAB.py [hunk 1]
```diff
@@ -0,0 +1,2724 @@
...
+        outputs_key_field='',
+        outputs=response,
+        raw_response=response
+    )
+
+    return command_results
+
+
+def test_module(client: Client) -> None:
+    # Test functions here
```
Comment (samuelFain) [inline Packs/WALLIXBastion/Integrations/WAB/WAB.py:2565]: Dont forget to complete the test_module function.

File: Packs/WALLIXBastion/Integrations/WAB/WAB.yml [hunk 2]
```diff
@@ -0,0 +1,4725 @@
+commonfields:
+  id: WALLIX Bastion
+  version: -1
+name: WALLIX Bastion
+display: WALLIX Bastion
+category: Authentication
+description: Centralized Control and Monitoring of Privileged Access to Sensitive Assets
+fromversion: 6.10.0
+tests:
+  - No test - todo
```
Comment (samuelFain) [inline Packs/WALLIXBastion/Integrations/WAB/WAB.yml:10]: Make sure do update the `tests` key if needed.

**Review by samuelFain — COMMENTED** [review]
@yoennb, Great Job!
Please see my comments, and make sure to merge from master and handle all the linter errors (flake8 etc.).
Also, don't forget to share the demo video with me when its available.

**Comment by yoennb** [comment]
Hello @samuelFain I have updated the PR according to your suggestions and added a few authentication handling code needed on our side. We are working on the video.

Also, by "merge from master" you mean rebase our branch on your master branch ?

**Review by samuelFain — COMMENTED** [review]
Hi @yoennb, please see these comments so that we may pass the build checks.
Feel free to share your video with me when its available 👍

File: Packs/WALLIXBastion/TestPlaybooks/WAB_test.yml
Comment (samuelFain) [inline Packs/WALLIXBastion/TestPlaybooks/WAB_test.yml:1]: Please add the missing `fromversion` field.

**Review by samuelFain — COMMENTED** [review]
@yoennb, great work!
Added a few minor comments to fix validation and linter errors.
Let me know when the demo video is ready, you can share it with me via my email sfainberg@paloaltonetworks.com.

File: Packs/WALLIXBastion/Integrations/WAB/WAB.py [hunk 3]
```diff
@@ -0,0 +1,2810 @@
...
+            headers["X-Auth-User"] = self._auth_user
+
+        client_err_handler = (
+            None if first_auth else self._raise_client_exc
+        )  # no AuthError if first_auth is True
+
+        try:
+            resp: Response = super()._http_request(
+                *args, **kwargs, resp_type="response", error_handler=client_err_handler
+            )
```
[the remark this answers is not in the document]
Comment (samuelFain) [reply Packs/WALLIXBastion/Integrations/WAB/WAB.py:44]: I don't mind if you believe it's not an issue with the code itself that can cause further problems.

**Comment by samuelFain** [comment]
Hi @yoennb,
This is a kindly reminder regarding the demo video.
If you prefer, we can schedule a meeting instead and do a live demo, whatever works best for you.

**Review by samuelFain — COMMENTED** [review]
@yoennb Great work, we are almost done here...
There are some last issues to address before we merge this PR:
- Please see my comment on updating the integration category in the yml file.
- There are some Flake8 and Mypy errors thats failing the linting checks, see them [here](https://app.circleci.com/pipelines/github/demisto/content/257774/workflows/502582ec-36c7-46b9-b253-b9f882c8938c/jobs/699568).

Let me know if something is unclear. 
BTY, the demo video was excellent!

**Comment by yoennb** [comment]
Thanks! I'm on it

**Comment by yoennb** [comment]
@samuelFain "contribution form filled label verification" --> should I fill the form again?

**Comment by samuelFain** [comment]
@yoennb, is it possible that the CLA is not filled yet? if so, please fill it as this PR can't be merged without it.
> Hi @yoennb, can you please [sign CLA](https://github.com/demisto/content/pull/25840#issuecomment-1508222422), and fill the [contribution form](https://github.com/demisto/content/pull/25840#issuecomment-1508224125)?

Also, we noticed the forked repo is named with `demisto` instead of the suggested `content` format.
This current situation is preventing us to finish the merging process.
Could you please manually rename the repo using this manual?
https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository

Please notify me if there are any questions.
Thanks!

**Comment by yoennb** [comment]
@samuelFain the CLA is already signed, I cannot sign it again. I tried filling the contribution form again just in case.

**Review by samuelFain — APPROVED** [review]

**Comment by samuelFain** [comment]
Merged 👍

**Comment by yoennb** [comment]
Excellent! Glad we managed to work this out. Should I request the renaming of our forked repo anyway?

**Pull request merged**